run_ibis_tests.py

```diff
@@ -503,9 +503,7 @@ def main():
                         arg_value = args_dict[pure_arg]
                         # correct filling of arguments with default values
                         if arg_value is not None:
```

Comment:
this was a quick fix from me, a correct one probably is below (requires testing):
```suggestion
                        if arg_value != parser.get_default(pure_arg):
```
as default value is not always `None`

Reply:
I do not understand why this change. Explain plz